app/models/application_record.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
+  
+  def total_money
```

Comment:
The only issue with this method here is that it will also apply to the `Trip` model which probably won't work or be useful.